packages/dx-react-scheduler/src/plugins/month-view.jsx

```diff
@@ -49,6 +53,10 @@ export class MonthView extends React.PureComponent {
       intervalCount, Date.now(),
     );
 
+    this.timeTableElementComputed = () => this.timeTable;
```

Comment:
There three are used only in this method and can be replaced with local variables.

```js
const timeTableElementComputed = () => this.timeTable;
...
```